New branch

TheShubhamJindal opened the pull request:
Cleaned path in Recommend.py from absolute to relative

TheShubhamJindal committed:
Cleaned the file path

TheShubhamJindal committed:
Cleaned the directory path

pratulyab commented:
Please clean the directory path in recommder/dump_dataset.py as well.
I've added you as collab

TheShubhamJindal commented:
I've cleaned the path in dump_dataset.py It was working fine on my system.
You also check it once, and if it is working fine, merge the newBranch to
the master.


On Mon, Jul 10, 2017 at 1:25 AM, Pratulya Bubna <notifications@github.com>
wrote:

> Please clean the directory path in recommder/dump_dataset.py as well.
> I've added you as collab
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/pratulyab/movie-recommender/pull/1#issuecomment-313950573>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARKHQje_xJ4tnzmd7z3cQxux92M9B7wNks5sMTAcgaJpZM4OSKim>
> .
>
